swatch-system-conduit/src/test/java/org/candlepin/subscriptions/conduit/admin/InternalOrganizationSyncResourceTest.java

```diff
@@ -68,4 +70,20 @@ void syncOrgShouldThrowInternalServerError() throws Exception {
   void syncAllOrgsShouldReturnSuccess() {
     assertEquals("Success", resource.syncFullOrgList().getStatus());
   }
+
+  @Test
+  void hasOrgInSyncListShouldReturnTrue() {
+    when(repo.existsById("123")).thenReturn(true);
+    assertTrue(
+        "Org ID expected to exist but does not",
+        resource.hasOrgInSyncList("123").getExistsInList());
+  }
+
+  @Test
+  void hasOrgInSyncListShouldReturnFalse() {
+    when(repo.existsById("123")).thenReturn(false);
+    assertFalse(
+        "Org ID expected to not exist but does",
```

Comment:
In this case shouldn't it be "Org Id doesn't exist"?

Reply:
Same as above comment but vice versa